low-level/framework/utils/salt_util.py

```diff
@@ -19,7 +19,7 @@
 """
 
 import configparser
-from cortx.utils.kv_store.kv_store_collection import PillarStore as PillarStorage
+from cortx.utils.kvstore.pillar import PillarStorage
```

Comment:
@mariyappanp this is reverting the code from main branch, which will break again. Basically this file change should not reflect in this patch!

Reply:
Oh my god. Sorry!. Thanks for finding this.
New commit has what is required.